modules/ui/sidebar.js

```diff
@@ -259,11 +273,16 @@ export function uiSidebar(context) {
                 inspector
                     .state('hide');
 
-            } else if (_wasRapid || _wasData || _wasNote || _wasQaItem) {
+            } else if (_wasRapid || _wasData || _wasNote || _wasQaItem || _wasMapRoulette) {
```

Comment:
Haha this code is getting pretty nutty.. 
Not your fault at all - just, I should really improve our sidebar UI!